roles/openshift_node/files/bootstrap.yml

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/ansible-playbook
```

Comment:
We should probably consider creating a separate role that can be applied post node setup that uses ansible to ship the bootstrap config into the same location.

I'd like to begin testing these roles in GCE by using the following playbook reorganization:

1. run openshift_node role with bootstrap = true on all nodes
2. run master configuration on masters
3. run openshift_node_bootstrap (role or playbook) which creates a kubeconfig and distributes to all nodes pointing back to the master

I can do this in a follow up, but this would follow the arc of generally moving *all* node installs to the bootstrap model unless the caller explicitly bypasses it.

Reply:
Let's discuss your ideas around this.